style.css

```diff
@@ -0,0 +1,327 @@
+.logo {
+  width: 76px;
+  height: 20px;
+  font-family: 'poppins', Arial, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #6070ff;
+  flex: none;
+  order: none;
+  flex-grow: 1;
+  border: rgb(247, 244, 244);
+}
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+#bttn {
+  font-family: 'Poppins', Arial, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #396df2;
+  padding: 12px;
+  background: #fff;
+  border: 1px solid #396df2;
+  border-radius: 12px;
+  margin: 15px 0;
+}
+
+#bttn:active {
+  background: #2e3de9;
+  color: #fff;
+}
+
+#bttn:hover {
+  background: #525ec4;
+  color: #fff;
+}
+
+.list {
+  list-style: none;
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  margin-bottom: 20px;
+  padding-right: 20px;
+}
```

Comment:
- [ ] As mentioned by the previous reviewer 📃 please add some spacing between the list items in the section of your work 👍🏽

### How it looks now
![image](https://user-images.githubusercontent.com/43172164/206844054-53b9fda1-7990-4705-a5be-6ba7c1894cc1.png)

- **_You can use the `gap: 5px;` in your .list class since you are using flexbox_**

Reply:
all done